tests/ha/cluster_state_mgmt.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Manage cluster stack
+#          Stop the cluster if pacemaker is active, otherwise start it.
+# Maintainer: Julien Adamek <jadamek@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils 'systemctl';
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+    if (systemctl('-q is-active pacemaker', ignore_failure => 1)) {
+        record_info("Start cluster", "Cluster is starting");
+        script_run "crm cluster start";
```

Comment:
Is this really needed? AFAICS the cluster is started after a reboot automatically, so the test module is actually only being used to stop the cluster (L29)

Reply:
Yes this is needed. In a QAM scenario, pacemaker is disabled at boot because we need to reboot the server twice.
One time after `zypper migration` and a second time after `post migration`, I add a reboot there because the maintenance package is installed in this module. I do not want the cluster to start between these two steps.
However, as I wrote in the summary, disable pacemaker at boot will not happen in the BV jobgroup.

Reply:
I understand. Follow up question though: why is the module scheduled only once in all the 15-SP1 & 15-SP2 QAM jobs linked in the PR? AFAICS module is scheduled only once to stop the cluster, and then cluster is being restarted by reboot. I do see `cluster_boot_mgmt` scheduled more than once, first time to disable pacemaker, and second time after first reboot to enable pacemaker. Is this is needed, why isn't it scheduled between the second `console_reboot` and `check_cluster_integrity`?

Reply:
As we said in private, the cluster could be started through `cluster_state_mgmt` but this is not used in this test. I did like that to have a consistency between `cluster_state `and `cluster_boot`.